http-api/src/v1/projects.rs

```diff
@@ -720,3 +720,402 @@ fn resolve_revisions(
         revisions,
     }
 }
+
+mod setup {
```

Comment:
This should be `#[cfg(test)]` probably?

Reply:
yup, was gonna move it into its separate file. see https://github.com/radicle-dev/radicle-client-services/pull/172/files#diff-191ca8c6d151a1a4c5243319a8395c8a17c03c6e257bf4d796b004f19a0bf469R9-R10